tracer/build/_build/PrepareRelease/GenerateIntegrationDefinitions.cs

```diff
@@ -595,7 +606,7 @@ public override bool Equals(object obj)
                 return Equals((CallTargetDefinitionSource)obj);
             }
 
-            public override int GetHashCode() => HashCode.Combine(TargetAssembly, TargetType, TargetMethod) + HashCode.Combine(TargetMinimumMajor, TargetMinimumMinor, TargetMinimumPatch,
+            public override int GetHashCode() => HashCode.Combine(IntegrationName, TargetAssembly, TargetType, TargetMethod) + HashCode.Combine(TargetMinimumMajor, TargetMinimumMinor, TargetMinimumPatch,
                                                                                                                                TargetMaximumMajor, TargetMaximumMinor, TargetMaximumPatch,
                                                                                                                                WrapperAssembly, WrapperType);
```

Comment:
Just noticed, we shouldn't be adding two hash codes I think.
```suggestion
            public override int GetHashCode()
            {
                var hash = new HashCode();
                hash.Add(IntegrationName);
                hash.Add(TargetAssembly);
                hash.Add(TargetType);
                hash.Add(TargetMethod);
                hash.Add(TargetMinimumMajor);
                hash.Add(TargetMinimumMinor);
                hash.Add(TargetMinimumPatch);
                hash.Add(TargetMaximumMajor);
                hash.Add(TargetMaximumMinor);
                hash.Add(TargetMaximumPatch);
                hash.Add(WrapperAssembly);
                hash.Add(WrapperType);
                return hash.ToHashCode();
            }
```

Reply:
Oh!, I'm pretty N00b with `HashCode`, Didn't know about that `Add` method.

Reply:
Me neither until just now 😄 Only necessary when combining more than 8 values

Reply:
TIL! Looks like `Add` and `Combine` were added in .NET 5. R# does something like in older versions (using `Span` as example):
```csharp
int hashCode = TraceId.GetHashCode();
hashCode = (hashCode * 397) ^ SpanId.GetHashCode();
hashCode = (hashCode * 397) ^ OperationName.GetHashCode();
// ...same for each member that should participate in equality operations...
return hashCode;
```